src/Engine/ProtoCore/ClassTable.cs

```diff
@@ -2,6 +2,7 @@
 using System.Collections.ObjectModel;

```

Comment:
This is being used on line 659, `Resources.kMultipleSymbolFound` references `Resources` which resides in `ProtoCore.Properties` namespace. The reason that line went into `master` without any failure was because `master` has another resource file `StringConstants.resx` (which has now been unified as `Resources.resx` in this pull request) under `src/Engine/ProtoCore`.